client/js/widgets/widget.js

```diff
@@ -984,20 +984,23 @@ export class Widget extends StateManaged {
       }
 
       if(a.func == 'FLIP') {
-        setDefaults(a, { count: 0, face: null, faceCyle: null, collection: 'DEFAULT' });
+        setDefaults(a, { count: 'all', face: null, faceCyle: null, collection: 'DEFAULT' });
+        if(a.count === 'all')
+          a.count = 999999;
```

Comment:
Perhaps since 999999 is used in so many places (including calls to the logging code) it should be defined in a constant somewhere?

Reply:
maybe you are right but I also don't see much advantage to changing it